client/src/pages/Contact.css

```diff
@@ -0,0 +1,220 @@
+/* body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+}
+
+.contact-background {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  background: url("../assets/images/turquie.jpeg") no-repeat center center /
+    cover;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+}
+
+.contact-background::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 0;
+}
+
+.contact-form-container {
+  position: relative;
+  background: white;
+  padding: 20px 30px;
+  border-radius: 10px;
+  background: rgba(255, 255, 255, 0.2);
+  box-shadow: 0 8px 32px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(10px);
+  z-index: 1;
+  width: 90%;
+  max-width: 400px;
+}
+
+.contact-form-container h2 {
+  margin-bottom: 15px;
+  color: #333;
+  text-align: center;
+}
+
+
+  .contact-form-container
+  label {
+  display: block;
+  margin-bottom: 5px;
+  font-weight: bold;
+}
+
+.contact-form-container input,
+.contact-form-container textarea,
+.contact-form-container button {
+  width: 100%;
+  padding: 10px;
+  margin-bottom: 15px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  font-size: 16px;
+}
+
+.contact-form-container button {
+  background-color: #00a676;
+  color: white;
+  border: none;
+  cursor: pointer;
+  font-size: 16px;
+  transition: background-color 0.3s;
+}
+
+.contact-form-container button:hover {
+  background-color: #00a676;
+}
+
+@media (max-width: 600px) {
+  .contact-form-container {
+    padding: 15px 20px;
+  }
+
+  .contact-form-container h2 {
+    font-size: 18px;
+  }
+}
+html,
+body {
+  margin: 0;
+  height: 100%;
+} */
```

Comment:
Pas de code mort

Reply:
https://media.tenor.com/6GYp9h_Nxg8AAAAM/lor-lord-of-the-rings.gif